sections/advanced/server-side-rendering.md

```diff
@@ -25,8 +25,12 @@ import { renderToString } from 'react-dom/server'
 import { ServerStyleSheet } from 'styled-components'
 
 const sheet = new ServerStyleSheet()
-const html = renderToString(sheet.collectStyles(<YourApp />))
-const styleTags = sheet.getStyleTags() // or sheet.getStyleElement();
+try {
+  const html = renderToString(sheet.collectStyles(<YourApp />))
+  const styleTags = sheet.getStyleTags() // or sheet.getStyleElement();
+} finally {
```

Comment:
Let's make sure we keep a `catch` statement in here, the optional catch binding feature hasn't proliferated yet AFAIK.

Reply:
It's also worth a comment that you must not call `.seal()` until AFTER trying to use `sheet.getStyleElement()` or it'll throw a different error.

Reply:
@probablyup I added `catch` statements to both examples and added an explanation that `.seal()` should be called after `sheet.getStyleElement()` or `sheet.getStyleTags()`